### PR TITLE
Update data.json with anon access example

### PR DESCRIPTION
**Pull request by fstn**

### CODE REVIEW

**Comment by fstn** [comment]
Add example that show how to give access to a topics for anon users

File: tinode-db/data.json [hunk 1]
```diff
@@ -244,6 +245,14 @@
...
    "topic": "*BF",
    "user": "bob"
+  },
+  {
+   "createdAt": "-126h",
+   "name": "support",
+   "owner": "frank",
+   "tags": ["support","public"],
+   "public": {"fn": "Support", "photo": "support-64.jpg", "type": "jpg"},
+   "access": {"anon": 64, "auth": 64}
```
Comment (or-else) [inline tinode-db/data.json]: This is a good idea but tinode-db/makedb.go must be updated too. Otherwise it's not going to be used.

**Comment by or-else** [comment]
What's the license of support.jpeg? Thanks!

**Comment by fstn** [comment]
https://pixabay.com/fr/adulte-d-affaires-homme-d-affaires-1260380/

File: tinode-db/data.json (hunk 1 above)
Comment (fstn) [reply tinode-db/data.json]: I commited changes to go file... but I don't know how to easily run it to test... what cmd line do you use to build and test makedb?

**Pull request merged**